packages/@react-spectrum/table/test/TableSizing.test.tsx

```diff
@@ -347,7 +347,8 @@ describe('TableViewSizing', function () {
         for (let row of bodyRows) {
           for (let [index, cell] of row.childNodes.entries()) {
             if (index === 0) {
-              expect((cell as HTMLElement).style.zIndex).toBe('2');
+              // TODO: This should be 2. Why did this change?
```

Comment:
zindex across columns is decrementing so that earlier columns render *over* later columns. otherwise the resizer would be partially covered by columns that follow